analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -232,24 +254,72 @@ class ComplexConditionsSingleNullable
 
     bool XorWithTrue(bool? b) => (true ^ b.HasValue) && b.Value;            // Noncompliant
     bool XorWithFalse(bool? b) => (false ^ b.HasValue) && b.Value;          // Compliant
+
+    void Reachability1(bool? b)
+    {
+        b = null;
```

Comment:
I don't have a preference. 
In this specific case, because there is a `b = null` assignment, both seem equally good to me.
I think it's important to test both approaches, to make sure that we are learning, in the same way, both on method parameters and local variables.
Apart from that, I don't have an opinion and I am open to changing it if you have arguments in favor of one or the other.